src/snek5000/info.py

```diff
@@ -41,12 +41,12 @@ def _init_root(self):
                     "scalar01",
                     "cvode",
                 ),
-                "par_sections_disabled": (
+                "par_sections_disabled": {
```

Comment:
Warning: this is the only real change in the code in this PR. But it's not nothing :slightly_smiling_face: It seems to works well.